tests/components/abode/test_light.py

```diff
@@ -41,13 +45,18 @@ async def test_attributes(hass: HomeAssistant) -> None:
     assert state.state == STATE_ON
     assert state.attributes.get(ATTR_BRIGHTNESS) == 204
     assert state.attributes.get(ATTR_RGB_COLOR) == (0, 63, 255)
-    assert state.attributes.get(ATTR_COLOR_TEMP) == 280
+    assert state.attributes.get(ATTR_COLOR_TEMP) is None
```

Comment:
The light was previously reporting both a color and a color temperature which is no longer allowed.